src/bin/commands/new-project/index.ts

```diff
@@ -144,23 +150,26 @@ export const newCdkProject = async (props: NewProjectProps): CliCommandResponse
     }
   );
 
+  CliUx.ux.action.start(chalk.yellow("Running lint check..."));
+  await commands.lint();
+  CliUx.ux.action.stop();
+
   CliUx.ux.action.start(chalk.yellow("Running tests..."));
-  await execute("./script/test", [], { cwd: config.cdkDir });
+  await commands.test();
   CliUx.ux.action.stop();
 
-  console.log(chalk.green("Summarising the created files"));
+  CliUx.ux.action.start(chalk.yellow("Running initial synthesis..."));
+  await commands.synth();
+  CliUx.ux.action.stop();
+
+  console.log(chalk.green("Success! Here's a summary of the created files:"));
   const tree = await execute("tree", ["-I 'node_modules|cdk.out'"], {
     cwd: config.cdkDir,
   });
   console.log(tree);
 
-  console.log("Project successfully created! Next steps:");
-
-  [
-    "Run ./script/diff to confirm there are no destructive changes (there should only be tag additions)",
-    "Update the repository's CI configuration to run ./script/ci",
-    "Raise a PR with these changes",
-  ].map((step) => console.log(`  - ${step}`));
+  const docsUrl = "https://github.com/guardian/cdk/tree/main/docs";
+  CliUx.ux.url(chalk.green(`Please see the docs (${docsUrl}) for next steps.`), docsUrl);
```

Comment:
This is a bit fancy 💅🏽 

![image](https://user-images.githubusercontent.com/836140/168775475-5263f1bb-e8ea-46c2-89da-7e07d3979c62.png)